packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -125,13 +125,28 @@ class DDSTableOfContents extends StableSelectorMixin(LitElement) {
     }
   }
 
+  /**
+   * Triggers focus on the component from parameter
+   *
+   * @param section The section receive focus
+   */
+  private _triggerFocus(section) {
+    const target = this.querySelector(`a[name="${section}"]`);
+    if (target) {
+      target.setAttribute('tabindex', '0');
+      (target as HTMLElement).focus({ preventScroll: true });
+      target.removeAttribute('tabindex');
+    }
```

Comment:
Can we simply add this code to `_handleUserInitiatedJump()` instead of introducing a new method?